lib/postgrex/types.ex

```diff
@@ -292,10 +292,12 @@ defmodule Postgrex.Types do
 
   defp fetch!(table, oid) do
     case :ets.lookup(table, oid) do
+      [{_, info, nil}] ->
+        raise ArgumentError, "no extension found for oid `#{oid}`: " <> inspect(info)
       [value] ->
         value
       [] ->
-        raise ArgumentError, message: "no extension found for oid `#{oid}`"
+        raise ArgumentError, "no extension found for oid `#{oid}`"
```

Comment:
There will still be many cases where we will hit this clause, since we only fetch the types from that database that match what `extension.matching` returns. Would it be worth it to also remove the big `WHERE` in the query to fetch all types instead?